scss/mixins/_large-logic.scss

```diff
@@ -8,20 +8,22 @@
   }
   .grid-zone.grid_#{$zone-index} {
       @for $sub-index from 1 through $columns-large {
-        .grid-blok.push_#{$sub-index} {
-          left: span($sub-index of $zone-index);
-        }
-        
-        .grid-blok.pull_#{$sub-index} {
-          left: -1 * span($sub-index of $zone-index);
-        }
-        
-        .grid-blok.prefix_#{$sub-index} {
-          margin-left: span($sub-index of $zone-index);
-        }
-        
-        .grid-blok.suffix_#{$sub-index} {
-          margin-right: span($sub-index of $zone-index);
+        @if $sub-index < $zone-index {
```

Comment:
Why not change the `@for` loop instead of adding an if() here?

`@for $sub-index from 1 through $zone-index`